### DISCUSSION TITLE
Field Validations

### DISCUSSION THREAD

**Discussion by luis3199correia** [OrchardCMS/OrchardCore#10822] [Q&A]
I've added a field validation in UpdateAsync(ContentFieldDisplayDriver<TextField>), and when importing JSON or package with content, the import process doesn't use this method.
Where can I add this field validation to validate fields in import process?

**Comment by deanmarcussen** [comment]
For importing, either via the api, or recipes you want to implement a ContentHandler for the part (there are no field handlers) or type that the field is attached to.

Example:
https://github.com/OrchardCMS/OrchardCore/blob/b9dfb4157b68f072f09a5f59f8153342f9788a0e/src/OrchardCore.Modules/OrchardCore.Autoroute/Handlers/AutoroutePartHandler.cs#L117

**Reply by luis3199correia** [reply]
Thanks for your help.
One more question, how can I remove/hide this notification?
https://github.com/OrchardCMS/OrchardCore/blob/b0c5e2be69689d99b9987e0396d9e8b7ab33b272/src/OrchardCore.Modules/OrchardCore.Contents/Controllers/AdminController.cs#L358
